src/sentry/api/helpers/group_index/validators/in_commit.py

```diff
@@ -7,7 +7,7 @@
 from sentry.models.repository import Repository
 
 
-class InCommitValidator(serializers.Serializer):
+class InCommitValidator(serializers.Serializer[Any]):
```

Comment:
looks like this isn't actually using the rest framework serializer bits -- probably should be `Serializer[object]` in that case